requirements-dev.in

```diff
@@ -0,0 +1,13 @@
+# This file specifies development dependencies. It is processed by pip-compile
+# to generate requirements-dev.txt. See the developer docs for more information.
+-e .
```

Comment:
As mentioned, you can remove this and rely on dev command invocations to set PYTHONPATH.  Easy now that you have a makefile.